src/esp/sensor/RedwoodNoiseModel.cu

```diff
@@ -134,7 +139,7 @@ struct CurandStates {
   curandState_t* devStates;
 
  private:
-  int n_blocks_;
+  int nStates_;
```

Comment:
```suggestion
  int nStates_ = 0;
```

Reply:
Makes sense as that's more consistent with my usage elsewhere.